src/Microsoft.IdentityModel.Xml/DelegatingXmlDictionaryWriter.cs

```diff
@@ -66,15 +67,25 @@ protected XmlDictionaryWriter InnerWriter
             set => _innerWriter = value ?? throw LogArgumentNullException(nameof(value));
         }
 
+        /// <summary>
+        /// Gets or sets the InternalWriter.
+        /// </summary>
+        /// <exception cref="ArgumentNullException"> if 'value' is null.</exception>
+        internal XmlDictionaryWriter InternalWriter
+        {
+            get => _internalWriter;
+            set => _internalWriter = value ?? throw LogArgumentNullException(nameof(value));
```

Comment:
Assuming the stack trace will still say which property was affected and thus this will be implicit? (meaning I think this is ok, but perhaps it makes more sense to do as you suggest)

Reply:
Correct, the stack trace will indicate the affected property. Looking at the [framework design guidance](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/using-standard-exception-types#argumentexception-argumentnullexception-and-argumentoutofrangeexception) the current approach looks fine:
>  DO use value for the name of the implicit value parameter of property setters.